openpype/hosts/nuke/plugins/publish/validate_rendered_frames.py

```diff
@@ -14,35 +14,44 @@ def get_instance(context, plugin):
         # Get the errored instances
         return get_errored_instances_from_context(context, plugin=plugin)
 
-    def repair_knob(self, instances, state):
+    def repair_knob(self, context, instances, state):
+        create_context = context.data["create_context"]
         for instance in instances:
-            node = instance.data["transientData"]["node"]
-            files_remove = [os.path.join(instance.data["outputDir"], f)
-                            for r in instance.data.get("representations", [])
-                            for f in r.get("files", [])
-                            ]
+            files_remove = [
+                os.path.join(instance.data["outputDir"], f_)
+                for r_ in instance.data.get("representations", [])
+                for f_ in r_.get("files", [])
```

Comment:
What if `"files"` is single file?

Reply:
In the future we should support single video file output, but at the moment we don't, so this should not happen.